zipline/data/resample.py

```diff
@@ -450,3 +453,48 @@ def volumes(self, assets, dt):
                     volumes.append(val)
                     continue
         return np.array(volumes)
+
+
+class MinuteResampleSessionBarReader(SessionBarReader):
+
+    def __init__(self, calendar, minute_bar_reader):
+        self._calendar = calendar
+        self._minute_bar_reader = minute_bar_reader
+
+    def _get_resampled(self, columns, start_dt, end_dt, assets):
+        minute_data = self._minute_bar_reader.load_raw_arrays(
```

Comment:
I can't remember - what type is `minute_data`?

Also, I was surprised that the minute_bar_reader doesn't need to know about the calendar.  Or was that already handled by whoever instantiated the minute_bar_reader?